pkg/config/setup/config.go

```diff
@@ -1136,6 +1136,7 @@ func fleet(config pkgconfigmodel.Setup) {
 	// Directory to store fleet policies
 	config.BindEnv("fleet_policies_dir")
 	config.SetDefault("fleet_layers", []string{})
+	config.BindEnv("config_id")
```

Comment:
Please add a default.

Reply:
Even if default is `""` ?

Reply:
@hush-hush Curious is `config.BindEnvAndSetDefault("config_id", "")` different or better than `config.BindEnv("config_id")` ?

Reply:
Yes, that way we have a type. We're trying to move toward a real schema for the configuration and untyped parameters are an issue.
It also ease the error handling around "is a settings exists or not" since it might change between each run of the agent depending on the env vars. Always having a value make the behavior consistent no matter the setup.